NORDIC Embedded Software/nrf52 Patient Monitoring Device/HRS_file.c

```diff
@@ -15,32 +15,35 @@ OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
 STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-#include <stdlib.h>
-#include <stdio.h>
+
 #include "HRS_file.h"
 #include "config.h"
 
 void MAX30105_config_parameter(uint8_t REGISTER, uint8_t value)
 {
     uint8_t reg[2] = {REGISTER, value};
-		
     ret_code_t err_code = nrf_drv_twi_tx(&MAX30105_twi, MAX30105_ADD, reg, sizeof(reg), false);
-    //printf("%d",err_code) ;
     APP_ERROR_CHECK(err_code);
 }
 void MAX30105_default_HRS_config(void)
 {
+    /*All magic numbers are register values enabling features
+    mentioned in comments, they are taken from MAX30105 documentation,
+    because they are being used only in that function they are not 
+    defined in .h file.	
+    */
+	
     MAX30105_config_parameter(MAX30105_FIFO_CONFIG, 112);
     // avreaging of 8 samples, enables rollover
     MAX30105_config_parameter(MAX30105_MODE_CONFIG, 7);
-    // ALL
+    // ALL diodes ON
     MAX30105_config_parameter(MAX30105_PARTICLE_SENSING_CONFIG, 39);
     // ADC:4096 SampleRate: 100 PulseWinteidith: 411
     MAX30105_config_parameter(MAX30105_LED1, 31);
-    //setting power on LEDs
     MAX30105_config_parameter(MAX30105_LED2, 31);
     MAX30105_config_parameter(MAX30105_LED3, 31);
```

Comment:
Could You describe better last argumentsvof ..._config_parameter() function? There are a lot magic numbers.